### PR TITLE
React 16.13

### PR DESCRIPTION
**Pull request by Marc-Andre-Rivet**
Companion to https://github.com/plotly/dash/pull/1145

### CODE REVIEW

File: src/dash-table/components/ControlledTable/index.tsx [hunk 1]
```diff
@@ -132,7 +132,7 @@ export default class ControlledTable extends PureComponent<ControlledTableProps>
         this.handleResize();
     }
 
-    componentWillMount() {
+    UNSAFE_componentWillMount() {
```
Comment (alexcjohnson) [inline src/dash-table/components/ControlledTable/index.tsx]: Is there a reason these listeners can't just go in `componentDidMount`?

File: src/dash-table/components/ControlledTable/index.tsx [hunk 2]
```diff
@@ -146,7 +146,7 @@ export default class ControlledTable extends PureComponent<ControlledTableProps>
         document.removeEventListener('paste', this.handlePaste);
     }
 
-    componentWillUpdate() {
+    UNSAFE_componentWillUpdate() {
```
Comment (alexcjohnson) [inline src/dash-table/components/ControlledTable/index.tsx]: Likewise, why not put this one in `componentDidUpdate`?

File: tests/integration/review_app/test_app_df_graph.py [hunk 3]
```diff
@@ -149,4 +155,10 @@ def update_graph(rows, selected_rows):
 
     dash_duo.start_server(app)
     dash_duo.wait_for_element("#waitfor")
+
+    # Wait for table
+    WebDriverWait(dash_duo.driver, _TIMEOUT).until(
+        EC.presence_of_element_located((By.CSS_SELECTOR, "#{}".format(IDS["table"])))
+    )
```
Comment (alexcjohnson) [inline tests/integration/review_app/test_app_df_graph.py]: Why not just `dash_duo.wait_for_element("#{}".format(IDS["TABLE"])`?

Comment (Marc-Andre-Rivet) [reply tests/integration/review_app/test_app_df_graph.py]: 🙈

File: tests/integration/review_app/test_app_df_graph.py [hunk 4]
```diff
@@ -7,9 +7,15 @@
 import dash_core_components as dcc
 import dash_html_components as html
 import dash_table
+import time
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
```
Comment (alexcjohnson) [inline tests/integration/review_app/test_app_df_graph.py]: Now you can get rid of these too, right? Which I'd have thought flake8 would have told you but perhaps it's not run on the test code...

Comment (Marc-Andre-Rivet) [reply tests/integration/review_app/test_app_df_graph.py]: You're right, it's apparently not run on the tests folder

Comment (Marc-Andre-Rivet) [reply tests/integration/review_app/test_app_df_graph.py]: https://github.com/plotly/dash-table/pull/713/commits/a77b1592c1caef144be2a64f380a05e3e5291c8e

File: .flake8 [hunk 5]
```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = E203, E266, E501, E731, W503
+ignore = C901, E203, E266, E501, E731, W503
```
Comment (Marc-Andre-Rivet) [inline .flake8]: Don't flag methods that are "too complex", we can decide that ourselves..

**Pull request merged**